policy/modules/kernel/kernel.te

```diff
@@ -390,10 +390,14 @@ ifdef(`init_systemd',`
 	')
 
 	optional_policy(`
+		dev_getattr_generic_usb_dev(kernel_t)
```

Comment:
The `getattr` is also provided by `dev_delete_generic_usb_dev(kernel_t)`; see `delete_chr_file_perms`.